### PR TITLE
Move the thumbnail scaling to ImageUtils

### PR DESCRIPTION
**Pull request by fcduarte**
Small refactoring on ImageUtils to have only one place that deals with thumbnail size/scale.

### CODE REVIEW

File: paparazzi/paparazzi/src/main/java/app/cash/paparazzi/internal/ImageUtils.kt [hunk 1]
```diff
@@ -312,6 +311,16 @@ internal object ImageUtils {
     }
   }
 
+  fun scaleToThumbnailSize(image: BufferedImage): BufferedImage {
+    val scale = getThumbnailScale(image)
+    return scale(image, scale, scale)
+  }
+
+  private fun getThumbnailScale(image: BufferedImage): Double {
+    val maxDimension = image.width.coerceAtLeast(image.height)
```
Comment (jrodbx) [inline paparazzi/paparazzi/src/main/java/app/cash/paparazzi/internal/ImageUtils.kt:320]:
```suggestion
    val maxDimension = max(image.width, image.height)
```

File: paparazzi/paparazzi/src/main/java/app/cash/paparazzi/internal/ImageUtils.kt [hunk 2]
```diff
@@ -312,6 +311,16 @@ internal object ImageUtils {
     }
   }
 
+  fun scaleToThumbnailSize(image: BufferedImage): BufferedImage {
+    val scale = getThumbnailScale(image)
+    return scale(image, scale, scale)
+  }
```
Comment (jrodbx) [inline paparazzi/paparazzi/src/main/java/app/cash/paparazzi/internal/ImageUtils.kt:317]: revert/inline?  there's only 1 usage of this.

**Review by jrodbx — APPROVED** [review]

File: paparazzi/paparazzi/src/main/java/app/cash/paparazzi/internal/ImageUtils.kt (hunk 2 above)
Comment (fcduarte) [reply paparazzi/paparazzi/src/main/java/app/cash/paparazzi/internal/ImageUtils.kt:317]: I'm a bit confused, inline the variable `scale` or the method `scaleToThumbnailSize `?

File: paparazzi/paparazzi/src/main/java/app/cash/paparazzi/internal/ImageUtils.kt [hunk 3]
```diff
@@ -33,6 +33,7 @@ import java.awt.image.BufferedImage.TYPE_INT_ARGB
 import java.io.File
 import java.io.File.separatorChar
 import java.io.IOException
+import java.lang.Integer.max
```
Comment (jrodbx) [inline paparazzi/paparazzi/src/main/java/app/cash/paparazzi/internal/ImageUtils.kt:36]: There's a kotlin jvm version of max to use.

File: paparazzi/paparazzi/src/main/java/app/cash/paparazzi/internal/ImageUtils.kt (hunk 2 above)
Comment (jrodbx) [reply paparazzi/paparazzi/src/main/java/app/cash/paparazzi/internal/ImageUtils.kt]: `scaleToThumbnailSize`

File: paparazzi/paparazzi/src/main/java/app/cash/paparazzi/internal/ImageUtils.kt (hunk 1 above)
Comment (jrodbx) [reply paparazzi/paparazzi/src/main/java/app/cash/paparazzi/internal/ImageUtils.kt]: sorry, not revert, but there's a Kotlin max to use here instead.

**Pull request merged**